addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10153,7 +10153,7 @@ msgctxt "#20173"
 msgid "FTP server"
 msgstr ""
 
-#: xbmc/network/GUIDialogNetworkSetup.cpp
+#: unused
```

Comment:
because some skins could be using them them though it's unlikely this one is used